tests/main/interfaces-hardware-random-control/task.yaml

```diff
@@ -1,6 +1,6 @@
 summary: Ensure that the hardware-random-control interface works.
 
-summary: |
+details: |
```

Comment:
nice catch, we should add something to the `run-checks` which validates all the task.yaml files to ensure they are valid yaml, since here clearly this was invalid YAML that we just never noticed since spread doesn't complain AFAICT